web/src/components/common/Card.tsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import CardImage from 'components/common/CardImage';
+import {Image} from 'interfaces';
+import styled from 'styled-components';
+
+interface CardContainerProps {
+  imgLeft?: boolean;
```

Comment:
I think `imgLeft` sounds like whether the image is on the left or on the right. I would use variable name like `orientation: landscape` or `isLandscape` instead. I think that would be clearer.

Reply:
Haha I was using `horizontal` but idk if it makes sense in this context so I changed to `imgLeft` 🤣. And I didn't wanna bring in an enum since we are only giving 2 options here hmm

Reply:
Ya I also thought of `horizontal` at first but I think `landscape` makes more sense in this context. I think `imgLeft` makes sense once you see the 2 layouts, only before you see that it's a bit unclear.

Reply:
+1 for 'imgLeft' seems not very clear. When I look at the interface, I'm not sure what does imgLeft=false mean

Reply:
@ifhuang So what word do you prefer? hahaha 😆